host/src/containers/GameSessionContainer.tsx

```diff
@@ -49,12 +49,13 @@ const GameSessionContainer = () => {
     let gameSessionSubscription: any | null = null;
     gameSessionSubscription = apiClient.subscribeUpdateGameSession(gameSessionId, response => {
       // only run the gametimer check on instances where the currentState changes (new screens)
-      if (gameSession && gameSession.currentState !== response.currentState) 
+      if (gameSession && gameSession.currentState !== response.currentState){
```

Comment:
nit: space before `{`. Same for the below changes in this file

Reply:
fixed